eng/tools/get-npm-tags.ps1

```diff
@@ -0,0 +1,87 @@
+param (
```

Comment:
Do you write unit tests or some form of automated tests for this code?

Reply:
I should definitely add unit test within script to get confidence about this script. Currently we will not have any automated way of testing it. But we can manually run those test cases to ensure it covers all scenarios.

Reply:
Added test cases